alphagov/paas-cf
# WIP [#153672095] Update cdn-broker with upstream changes

## What

This PR updates the cdn-broker release to the latest version.

We are updating the CDN broker because there is a bug in the certificate renewal process which causes certificates renewed multiple times unnecessarily. The upstream changes from 18F fix this issue.

Also we want to update so we get other fixes and updates from upstream.

❗️ This PR currently contains a temporary commit updating the release to a dev version. Please replace it when https://github.com/alphagov/paas-cdn-broker-boshrelease/pull/8 gets merged.

## How to review

1. Update your pipeline from this branch and update Cloud Foundry
2. Create a cdn-route service and test it if it works as expected
3. Delete the cdn-route service

TODO: Exact testing instructions

## Who can review

Not @camelpunch or @bandesz
